eclair-core/src/main/scala/fr/acinq/eclair/payment/receive/MultiPartHandler.scala

```diff
@@ -339,8 +339,8 @@ object MultiPartHandler {
                 Future.sequence(r.routes.map(route => {
                   val pathId = randomBytes32()
                   val dummyHops = route.dummyHops.map(h => {
-                    val edge = Invoice.BasicEdge(nodeParams.nodeId, nodeParams.nodeId, ShortChannelId.toSelf, h.feeBase, h.feeProportionalMillionths, h.cltvExpiryDelta)
-                    ChannelHop(edge.shortChannelId, edge.sourceNodeId, edge.targetNodeId, ChannelRelayParams.FromHint(edge))
+                    val edge = Invoice.ExtraEdge(nodeParams.nodeId, nodeParams.nodeId, ShortChannelId.toSelf, h.feeBase, h.feeProportionalMillionths, h.cltvExpiryDelta, 1 msat, None)
```

Comment:
Not really, it was actually hidden in the conversion from a `BasicEdge` to a `GraphEdge`...the values here make sense for a dummy hop: we allow any HTLC size, so min should be `1 msat` (because a 0-value HTLC is not spec-compliant) and max should be `None`, does it make sense?

Reply:
A comment is still nice though? I also named args in my suggestion.

Reply:
Done in https://github.com/ACINQ/eclair/pull/2480/commits/bdbeda352a8f0860dc62ea8523232491d6031cb1 and https://github.com/ACINQ/eclair/pull/2480/commits/9627acab2b69d10dff80c51bf0af65ad59893268